packages/client-sdk-nodejs-compression/README.template.md

```diff
@@ -14,19 +14,37 @@ npm install @gomomento/sdk-compression
 Then, you can configure the Momento cache client to enable compression:
 
 ```javascript
-{% include "./examples/nodejs/compression/configuration.ts" %}
+configuration.withCompressionStrategy({
+  compressorFactory: CompressorFactory.default(),
+  compressionLevel: CompressionLevel.SmallestSize,
+})
+
```

Comment:
Is our convention to inline the code like this or to refer to files like before to ensure they are still valid?

Reply:
The compiler would not compile keeping just the code for configuration, sets an gets in separate files. Also, it can get confusing for someone to look inside examples and look at those raw files?

Reply:
Let's revisit with @cprice404 later if this is better handled with the API snippets injector.